src/config.js

```diff
@@ -23,6 +23,11 @@ const defaults = {
     width: 256,
     height: -1
   },
+  metadata: {
```

Comment:
skip the enhanced true/false, Just check if `metadata != null`.

Reply:
Wouldn't that mean I cant define default settings here though, unless you want it to default to enabled?

Reply:
default to enabled